src/webviews/extension-side/dataviewer/constants.ts

```diff
@@ -0,0 +1,6 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+'use strict';
+
+export const pandasMinimumVersionSupported = '0.20.0';
```

Comment:
I'd change this to be more descriptive or add comments.
```suggestion
export const pandasMinimumVersionSupportedByVariableViewer = '0.20.0';
```